### PR TITLE
Enhance the warning message of RRDSigarDataStore-UnsupportedOperationException

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues

1. MAJOR [RRDSigarDataStore.java#L193](https://github.com/ow2-proactive/scheduling/blob/ba8c563fb80c49e7c10ede15a1755518f84929f1/rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RRDSigarDataStore.java#L193): Either log or rethrow this exception. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1166)
1. MINOR [RRDSigarDataStore.java#L190](https://github.com/ow2-proactive/scheduling/blob/ba8c563fb80c49e7c10ede15a1755518f84929f1/rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RRDSigarDataStore.java#L190): Define a constant instead of duplicating this literal " for object " 3 times. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1192)

File: rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RRDSigarDataStore.java [hunk 1]
```diff
@@ -190,7 +190,10 @@ void sample(RrdDb dataBase, long timeInMs) throws IOException {
                     logger.trace("Cannot read attribute " + attrName + " for object " + objectName + ": " +
                                  e.getMessage());
                 }
-
+            } catch (UnsupportedOperationException e) {
+                if (logger.isTraceEnabled()) {
```
Comment (fviale) [inline rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RRDSigarDataStore.java]: I would do something different, if logger.isDebugEnabled() then print this message (in warning)
if logger.isTraceEnabled() then print the full exception

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [RRDSigarDataStore.java#L190](https://github.com/ow2-proactive/scheduling/blob/d53cbd9d7b981974d25d08f04af78c4782fa8536/rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RRDSigarDataStore.java#L190): Define a constant instead of duplicating this literal " for object " 4 times. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1192)

**Comment by fviale** [comment]
Retest this please

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [RRDSigarDataStore.java#L190](https://github.com/ow2-proactive/scheduling/blob/d4b161713aa7ee44997a12c5c0913b8819662ec8/rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RRDSigarDataStore.java#L190): Define a constant instead of duplicating this literal " for object " 4 times. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1192)

**Pull request merged**